lambdas/static-asset-uploader/index.js

```diff
@@ -195,11 +198,11 @@ function createCatalogDirectory(staticBucketName) {
  */
 async function createSdkGenerationFile(staticBucketName) {
     return await exports.s3.headObject({ Bucket: staticBucketName, Key: 'sdkGeneration.json' }).promise()
-        .catch((error) => {
-            console.error(error)
+        .catch(async (_) => {
```

Comment:
What's the `_` for?

Reply:
Oops, that refers to the unused `error` argument. It should be named `_error`.